### DISCUSSION TITLE
Glow scores make me feel strange

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#26880] [Ideas] [answered]
I have a strange feeling about these glow scores, especially those with low scores. I think we should reflect them in a different way... For example, only 'Not Break' scores are glow?

I think, the reason for this feeling is because it is blue glow (S??)

Additionally, sometimes feeling crowded, perhaps the glow effect should be reduced?

**Comment by Stoppedpuma** [comment, marked as answer]
The idea for FC scores seems to already be plan with the upcoming song select update, the glow is being replaced with gradients in the upcoming designs. See "Alternative" in the image below.
![clipboard](https://github.com/ppy/osu/assets/58333920/020c5df7-0381-4709-bcdc-cc2d7cec28b2)
